sdk/identity/Azure.Identity/tests/DefaultAzureCredentialLiveTests.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Core.Tests;
+using Azure.Identity.Tests.Mock;
+using NUnit.Framework;
+
+namespace Azure.Identity.Tests
+{
+    public class DefaultAzureCredentialLiveTests : RecordedTestBase<IdentityTestEnvironment>
+    {
+        private const string ExpectedServiceName = "VS Code Azure";
+
+        public DefaultAzureCredentialLiveTests(bool isAsync) : base(isAsync)
+        {
+            Matcher.ExcludeHeaders.Add("Content-Length");
+            Matcher.ExcludeHeaders.Add("client-request-id");
+            Matcher.ExcludeHeaders.Add("x-client-OS");
+            Matcher.ExcludeHeaders.Add("x-client-SKU");
+            Matcher.ExcludeHeaders.Add("x-client-CPU");
+
+            Sanitizer = new IdentityRecordedTestSanitizer();
+            TestDiagnostics = false;
+        }
+
+        [Test]
+        public async Task DefaultAzureCredential_UseVisualStudioCredential()
+        {
+            var options = Recording.InstrumentClientOptions(new DefaultAzureCredentialOptions
+            {
+                ExcludeEnvironmentCredential = true,
+                ExcludeInteractiveBrowserCredential = true,
+                ExcludeSharedTokenCacheCredential = true,
```

Comment:
These three types of credentials are temporary excluded. `EnvironmentCredential` can be tested live but it requires some additional code changes to pass `TokenCredentialOptions` into underlying credentials.